src/regional_settings.cpp

```diff
@@ -604,13 +662,13 @@ void load_region_settings( const JsonObject &jo )
 
     load_overmap_feature_flag_settings( jo, new_region.overmap_feature_flag, strict, false );
 
-    load_overmap_forest_settings( jo, new_region.overmap_forest, strict, false );
+    optional( jo, false, "overmap_forest_settings", new_region.overmap_forest, bogus_forest_id );
 
-    load_overmap_lake_settings( jo, new_region.overmap_lake, strict, false );
+    optional( jo, false, "overmap_ravine_settings", new_region.overmap_ravine, bogus_ravine_id );
 
-    load_overmap_ocean_settings( jo, new_region.overmap_ocean, strict, false );
+    optional( jo, false, "overmap_ocean_settings", new_region.overmap_ocean, bogus_ocean_id );
```

Comment:
Then, when the settings aren't specified, they just load the bogus id, which is `""`, and doesn't correspond to any actually loaded JSON object.